server/pairing/server.go

```diff
@@ -245,6 +245,7 @@ func MakeFullReceiverServer(backend *api.GethStatusBackend, config *ReceiverServ
 		return nil, err
 	}
 
+	// ignore err because we don't care if there is no active account
```

Comment:
hi @qfrank : did you leave this comment by mistake?

Reply:
Hi @siddarthkay , i'd say no, it should allow no active account here

Reply:
yes all good, I just saw a comment with no code change and assumed you either forgot to change code in this file or added the comment by mistake :D 
Thanks for explaining!